prog/postgres/postgres_timeline_nexus.rb

```diff
@@ -44,6 +44,16 @@ def before_run
       hop_take_backup
     end
 
+    # For the purpose of missing backup pages, we act like the very first backup
+    # is taken at the creation, which ensures that we would get a page if and only
+    # if no backup is taken for 2 days.
+    latest_backup_completed_at = postgres_timeline.backups.map(&:last_modified).max || created_at
+    if postgres_timeline.leader && latest_backup_completed_at < Time.now - 2 * 24 * 60 * 60 # 2 days
+      Prog::PageNexus.assemble("Missing backup at #{postgres_timeline}!", [postgres_timeline.ubid], "MissingBackup", postgres_timeline.id)
```

Comment:
Since we have the ubid in the 'related_resources' parameter, what are your thoughts on putting the resource name to the summary? This could help on-call engineers, as they would need to remember the user-friendly resource name rather than the 'ubid' for recurring incidents.

Reply:
I think it would be nice from on-call perspective. But we send this to PagerDuty and I'm not sure if customer defined names count as PII. We use PagerDuty's EU service region, so it might be OK from GDPR perspective, but I'm not expert on this.